frontend/src/app/components/wp-table/configuration-modal/tabs/display-settings-tab.component.html

```diff
@@ -1,63 +1,64 @@
 <div>
   <form>
+    <p>Display work packages as</p>
```

Comment:
Please translate. To avoid missing these kinds of strings, *always* use `[textContent]="text.myTextPropery"` even if you don't have it in your js-en.yml yet. It will raise an error from Angular to help you spot these.